GitUI/BranchTreePanel/RepoObjectsTree.ContextActions.cs

```diff
@@ -281,8 +281,8 @@ private void RegisterContextActions()
             _tagNodeMenuItems = new TagMenuItems<TagNode>(this);
             AddContextMenuItems(menuTag, _tagNodeMenuItems.Select(s => s.Item));
 
-            RegisterClick(mnubtnCollapseAll, () => treeMain.CollapseAll());
-            RegisterClick(mnubtnExpandAll, () => treeMain.ExpandAll());
+            RegisterClick(mnubtnCollapseAll, () => treeMain.SelectedNode.Collapse());
+            RegisterClick(mnubtnExpandAll, () => treeMain.SelectedNode.ExpandAll());
```

Comment:
I guess we need a check for `null`: One can disable all root nodes, e.g. temporarily when switching between single categories displayed.

Reply:
You're right. Fixed.